.github/workflows/gen_docs.yaml

```diff
@@ -0,0 +1,18 @@
+name: Generate Documenation Pages
+on:
+  push:
+    branches:
+      - main
+jobs:
+  generate:
+    if: ${{ github.repository_owner == 'bazelbuild' }}
+    name: Generate Docs
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install bazelisk
```

Comment:
Also, seems Bazelisk is available on Ubuntu-2004 machines: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md

This step could probably be removed